src/utils/getVisibleDays.js

```diff
@@ -5,6 +5,7 @@ export default function getVisibleDays(
   month,
   numberOfMonths,
   enableOutsideDays,
+  firstDayOfWeek = moment.localeData().firstDayOfWeek(),
```

Comment:
this is a breaking change to this publicly accessible function; please move the new argument to the end so it can be semver-minor.